### PR TITLE
upgraded to use the latest version of pogo

### PR DESCRIPTION
**Pull request by dereke**
some tests ignored as I am not sure how/what they are supposed to do

### CODE REVIEW

File: test/qo-psSpec.pogo [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-            rimraf! 'test/a'
+            mkdirp!('test/a/b/c', ^)
+            expect(ps.exec!('pwd', cwd: 'test/a/b/c')).to.match r/\/a\/b\/c\n$/
+            rimraf!('test/a', ^)
 
-        it 'executes a process in a given directory'
-            mkdirp! 'test/a/b/c'
-            ps.exec! 'ls' '-al' (cwd: '/').should.match r/\/a\/b\/c\n$/
-            rimraf! 'test/a'
+        xit 'executes a process in a given directory'
```
Comment (refractalize) [inline test/qo-psSpec.pogo:16]: hi @dereke any reason these were skipped?

**Comment by dereke** [comment]
yeah as noted on my comment I wasn't sure what they were suppsed to do.
I think that `ls -al` may have different output on linux/macos because those tests made no sense with what I was seeing

**Comment by refractalize** [comment]
oh ok, sorry didn't read it :)

**Comment by refractalize** [comment]
cheers @dereke, in v0.1.0

**Pull request merged**